src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -385,14 +385,7 @@ const SAVED_SEARCH_ALERT_EDIT_FORM_QUERY = graphql`
     artworksConnection(
       first: 0
       artistIDs: $artistIds
-      aggregations: [
-        ARTIST
-        LOCATION_CITY
-        MATERIALS_TERMS
-        MEDIUM
-        PARTNER
-        COLOR
-      ]
+      aggregations: [LOCATION_CITY, MATERIALS_TERMS, MEDIUM, PARTNER, COLOR]
```

Comment:
We don't need to use data from the `ARTIST` aggregation here, since we get information about artists [here](https://github.com/artsy/force/blob/3742a208af183454173f22e7d19511d80773ce0d/src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx#L344-L354)